### ISSUE TITLE
Effects are always the same

### ISSUE THREAD

**Issue by gitshell007** [lexrus/LTMorphingLabel#101]
I try to change different events but always works with the same effect:

 let morphingEffect: LTMorphingEffect = .pixelate
  cell.lblLastMessage.morphingEffect = morphingEffect

or changing rawValue

if let effect = LTMorphingEffect(rawValue: 2) {
         cell.lblLastMessage.morphingEffect = effect
}

**Comment by lexrus** [comment]
Please update to the latest release 0.5.5 and try again.

**Comment by kershner** [comment]
I also have this problem.  No matter what value I choose for the label's **.morphingEffect** it looks to be using the default **Scale**

Here's how I init the label:
`self.animatedLabel1 = LTMorphingLabel()`
`self.animatedLabel1.morphingEffect = .fall`

And the animation is always Scale.

I got this result on the **0.5.6** release, and then I also tried the **0.5.5** release per this thread.

Any idea what the issue might be?  Thanks for your great work on this Pod.
